.mega-linter.yml

```diff
@@ -2,3 +2,6 @@ ENABLE_LINTERS:
   - HTML_DJLINT
   - MARKDOWN_MARKDOWNLINT
   - EDITORCONFIG_EDITORCONFIG_CHECKER
+
+# We use the Javascript flavor, which does not include all of the linters.
+FAIL_IF_MISSING_LINTER_IN_FLAVOR: true
```

Comment:
```suggestion
```
It's better to have it fail if we use a linter that's not in this flavour

Reply:
I'm confused. Isn't the setting you're removing explicitly that? Otherwise, it _won't fail_ when we add linters that aren't present, which is the default. This makes sure we get errors and know if we're adding a linter not a part of the flavor.

Reply:
:facepalm: I read a negation somewhere in that, never mind